catalog/tests/test_views.py

```diff
@@ -0,0 +1,121 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+from catalog.models import Cook, Dish, DishType
+
+
+class ViewTests(TestCase):
```

Comment:
Brakuje testów z negatywnymi wynikami. Czyli co zadzieje się z aplikacją (jakie błędy polecą), gdy ktoś usilnie będzie probował ją popuść. Dla zapytań typu GET można to pominąć, bo tutaj użytkownik nie wiele może zdziałać, ale w zapytaniach typu POST już można namieszać.